warehouse/transformer/transformer_test.go

```diff
@@ -0,0 +1,753 @@
+									"sent_at":                  "2021-09-01T00:00:00.000Z",
+									"timestamp":                "2021-09-01T00:00:00.000Z",
+									"user_id":                  "userId",
+								},
+								lo.SliceToMap(
+									lo.RepeatBy(500, func(index int) string {
+										return strconv.Itoa(index)
+									}), func(item string) (string, any) {
+										return fmt.Sprintf(`context_column_%s`, item), fmt.Sprintf(`value_%s`, item)
+									},
+								),
+							),
+							"metadata": map[string]any{
+								"columns": lo.Assign(
+									map[string]any{
+										"anonymous_id":             "string",
+										"channel":                  "string",
+										"context_destination_id":   "string",
+										"context_destination_type": "string",
+										"context_ip":               "string",
+										"context_passed_ip":        "string",
+										"context_request_ip":       "string",
+										"context_source_id":        "string",
+										"context_source_type":      "string",
+										"context_traits_email":     "string",
+										"context_traits_logins":    "int",
+										"context_traits_name":      "string",
+										"event":                    "string",
+										"event_text":               "string",
+										"id":                       "string",
+										"original_timestamp":       "datetime",
+										"product_id":               "string",
+										"rating":                   "int",
+										"received_at":              "datetime",
+										"review_body":              "string",
+										"review_id":                "string",
+										"sent_at":                  "datetime",
+										"timestamp":                "datetime",
+										"user_id":                  "string",
+										"uuid_ts":                  "datetime",
+									},
+									lo.SliceToMap(
+										lo.RepeatBy(500, func(index int) string {
+											return strconv.Itoa(index)
+										}), func(item string) (string, any) {
+											return fmt.Sprintf(`context_column_%s`, item), "string"
+										},
+									),
+								),
+								"receivedAt": "2021-09-01T00:00:00.000Z",
+								"table":      "event",
+							},
+							"userId": "",
+						},
+						Metadata: ptrans.Metadata{
+							EventType:       "track",
+							DestinationType: "POSTGRES",
+							ReceivedAt:      "2021-09-01T00:00:00.000Z",
+							SourceID:        "sourceID",
+							DestinationID:   "destinationID",
+							SourceType:      "sourceType",
+							MessageID:       "messageId",
+						},
+						StatusCode: http.StatusOK,
+					},
+				},
+			},
+		},
+	}
+
+	for _, tc := range testsCases {
+		t.Run(tc.name, func(t *testing.T) {
+			pool, err := dockertest.NewPool("")
+			require.NoError(t, err)
+
+			opts := lo.Map(tc.envOverride, func(item string, index int) transformertest.Option {
+				return transformertest.WithEnv(item)
+			})
+			opts = append(opts, transformertest.WithRepository("rudderstack/develop-rudder-transformer"))
```

Comment:
how do you access this image? 
```
            	            	failed to pull image: API error (404): pull access denied for rudderstack/develop-rudder-transformer, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Reply:
`develop-rudder-transformer` is in rudderlabs and I was testing a [test case](https://github.com/rudderlabs/rudder-transformer/pull/3806) locally. I have commented the test case until the latest transformer image has the changes.